Examples/S3EventNotifier/README.md

```diff
@@ -76,4 +82,7 @@ This will:
  - configure the bucket to send `s3:ObjectCreated:*` events to the Lambda function named `S3EventNotifier`;
  - upload a file named `testfile.txt` to the bucket.
 
-Replace `<REGION>` with the region where you deployed the Lambda function and `<YOUR_ACCOUNT_ID>` with your actual AWS account ID.
+Replace `my-test-bucket` with your bucket name (bucket names are unique globaly and this one is already taken). Also replace `<REGION>` with the region where you deployed the Lambda function and `<YOUR_ACCOUNT_ID>` with your actual AWS account ID.
+
+[!IMPORTANT]
+The Lambda function and the S3 bucket must be located in the same AWS Region. Adjust the code above according to your closest AWS Region.
```

Comment:
This may be unnecessary if we have the one above already? But I'm not precious about it. If you do end up leaving it, this should use block quotes too

Reply:
https://github.com/orgs/community/discussions/16925

Reply:
I made it on purpose - it so important and I'm sure so many will not read it :-)